### PR TITLE
release litmus 3.5.0

### PR DESCRIPTION
**Pull request by jouve**
#### What this PR does / why we need it:

release litmus 3.5.0 as a chart

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by Jasstkn — APPROVED** [review]

File: charts/litmus/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "3.1.0"
+appVersion: "3.3.0"
 description: A Helm chart to install ChaosCenter
 name: litmus
-version: 3.1.0
+version: 3.3.0
 kubeVersion: ">=1.16.0-0"
```
Comment (Saranya-jena) [inline charts/litmus/Chart.yaml:6]: Can we change the version to latest  3.4.0?

File: charts/litmus/values.yaml [hunk 2]
```diff
@@ -223,23 +223,23 @@ portal:
...
-        LITMUS_CHAOS_EXPORTER_IMAGE: "chaos-exporter:3.1.0"
+        LITMUS_CHAOS_OPERATOR_IMAGE: "chaos-operator:3.4.0"
+        LITMUS_CHAOS_RUNNER_IMAGE: "chaos-runner:3.4.0"
+        LITMUS_CHAOS_EXPORTER_IMAGE: "chaos-exporter:3.4.0"
       genericEnv:
         TLS_SECRET_NAME: ""
         TLS_CERT_64: ""
         CONTAINER_RUNTIME_EXECUTOR: "k8sapi"
-        DEFAULT_HUB_BRANCH_NAME: "v3.1.x"
+        DEFAULT_HUB_BRANCH_NAME: "v3.3.x"
```
Comment (Jonsy13) [inline charts/litmus/values.yaml:237]:
```suggestion
        DEFAULT_HUB_BRANCH_NAME: "v3.4.x"
```

File: charts/litmus/README.md [hunk 3]
```diff
@@ -156,24 +156,24 @@ We separated service configuration from `portal.server.service` to `portal.serve
 | portal.server.customLabels | object | `{}` |  |
 | portal.server.graphqlServer.genericEnv.CHAOS_CENTER_UI_ENDPOINT | string | `""` |  |
 | portal.server.graphqlServer.genericEnv.CONTAINER_RUNTIME_EXECUTOR | string | `"k8sapi"` |  |
-| portal.server.graphqlServer.genericEnv.DEFAULT_HUB_BRANCH_NAME | string | `"v3.1.x"` |  |
-| portal.server.graphqlServer.genericEnv.INFRA_COMPATIBLE_VERSIONS | string | `"[\"3.1.0\"]"` |  |
+| portal.server.graphqlServer.genericEnv.DEFAULT_HUB_BRANCH_NAME | string | `"v3.3.x"` |  |
```
Comment (Jonsy13) [inline charts/litmus/README.md:159]: Once updated values.yml, we need to generate docs as well.

File: charts/litmus/values.yaml [hunk 4]
```diff
@@ -223,23 +223,23 @@ portal:
...
-        LITMUS_CHAOS_EXPORTER_IMAGE: "chaos-exporter:3.1.0"
+        LITMUS_CHAOS_OPERATOR_IMAGE: "chaos-operator:3.5.0"
+        LITMUS_CHAOS_RUNNER_IMAGE: "chaos-runner:3.5.0"
+        LITMUS_CHAOS_EXPORTER_IMAGE: "chaos-exporter:3.5.0"
       genericEnv:
         TLS_SECRET_NAME: ""
         TLS_CERT_64: ""
         CONTAINER_RUNTIME_EXECUTOR: "k8sapi"
-        DEFAULT_HUB_BRANCH_NAME: "v3.1.x"
+        DEFAULT_HUB_BRANCH_NAME: "v3.4.x"
```
Comment (Jonsy13) [inline charts/litmus/values.yaml:237]: Can you also update this to 3.5.x

**Comment by Jonsy13** [comment]
Hi @jouve Please also run `helmdocs`. check-docs-update is failing due to same reason.

**Comment by jouve** [comment]
ran with helm-docs 1.12.0 instead of 1.11.0 ... let me edit this

**Review by ispeakc0de — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Comment by Jonsy13** [comment]
Thanks @jouve for making the changes!

**Pull request merged**